dashboard/app/controllers/application_controller.rb

```diff
@@ -208,7 +208,13 @@ def set_locale_cookie(locale)
     cookies[:language_] = {value: locale, domain: :all, expires: 10.years.from_now}
   end
 
+  def require_english_in_levelbuilder_mode
+    redirect_to '/', flash: {alert: 'Editing on levelbuilder is only supported in English (en-US locale).'} unless locale == :'en-US'
+  end
+
   def require_levelbuilder_mode
+    require_english_in_levelbuilder_mode
```

Comment:
thank you for pushing for this more general approach, Dani! I think this is quite likely to save us from another surprise unwanted translation somewhere besides the script and level edit pages.

Reply:
+1000